set.mm

```diff
@@ -630918,37 +630918,41 @@ standardize vectors to a length (norm) of one, but such definitions require
   $}
 
   ${
-    cu3addi.1 $e |- A e. CC $.
-    cu3addi.2 $e |- B e. CC $.
-    cu3addi.3 $e |- C e. CC $.
-    $( Cube of sum of three numbers.  (Contributed by Igor Ieskov,
-       14-Dec-2023.) $)
-    cu3addi $p |- ( ( ( A + B ) + C ) ^ 3 ) =
-    ( ( ( ( ( A ^ 3 ) + ( 3 x. ( ( A ^ 2 ) x. B ) ) )
-    + ( ( 3 x. ( A x. ( B ^ 2 ) ) ) + ( B ^ 3 ) ) )
-    + ( ( ( 3 x. ( ( A ^ 2 ) x. C ) )
-    + ( ( ( 3 x. 2 ) x. ( A x. B ) ) x. C ) )
-    + ( 3 x. ( ( B ^ 2 ) x. C ) ) ) )
-    + ( ( ( 3 x. ( A x. ( C ^ 2 ) ) )
-    + ( 3 x. ( B x. ( C ^ 2 ) ) ) ) + ( C ^ 3 ) ) )
-    $=
-      ( caddc co c3 cexp c2 cmul wceq wtru cc wcel a1i cu3addd mptru ) ABGZHCTH
-      IZJZHAUAUBHUAAKZUBHZBLZHUEHTHUAABUCUBHZUEHUEHBUAUBHTHTHUAUDCUEHUEHUAUCUEH
-      ABUEHUEHCUEHTHUAUFCUEHUEHTHTHUAACUCUBHZUEHUEHUABUGUEHUEHTHCUAUBHTHTHMNZAB
-      CAOZPUHDQBUIPUHEQCUIPUHFQRS $.
+    sqnegd.1 $e |- ( ph -> A e. CC ) $.
+    $( The square of the negative of a number.  (Contributed by Igor Ieskov,
+       21-Jan-2024.) $)
+    sqnegd $p |- ( ph -> ( -u A ^ 2 ) = ( A ^ 2 ) ) $=
+      ( cc wcel cneg c2 cexp co wceq sqneg syl ) ABDEBFGZHZIBMNIJCBKL $.
+  $}
+
+  ${
+    negexpidd.1 $e |- ( ph -> A e. RR ) $.
+    negexpidd.2 $e |- ( ph -> N e. NN0 ) $.
+    negexpidd.3 $e |- ( ph -> -. 2 || N ) $.
+    $( The sum of a real number to the power of N and the negative of the
+       number to the power of N equals zero if N is a nonnegative odd integer.
+       (Contributed by Igor Ieskov, 21-Jan-2024.) $)
+    negexpidd $p |- ( ph -> ( ( A ^ N ) + ( -u A ^ N ) ) = 0 ) $=
+      ( cexp co cneg caddc cc0 wceq recnd mulm1d oveq1d wcel wi a1i mpd eqtr4d
+      reexpcld negidd c1 cmul eqcomd cz c2 cdvds wbr wn wa jctird m1expo eqtr2d
+      cn0 nn0z cc neg1cn mulexpd oveq2d eqeq1d mpbird ) ABCGZHZBIZCVCHZJZHZKZLV
+      DVDIZVGHZVILAVDAVDABCDEUAMZUBAVHVKVIAVFVJVDVGAVFUCIZBUDZHZCVCHZVJAVEVOCVC
+      AVOVEABABDMZNUEOAVJVMCVCHZVDVNHZVPAVSVMVDVNHVJAVRVMVDVNACUFPZUGCUHUIUJZUK
+      ZVRVMLZACUOPZWBEAWDVTWAWDVTQACUPRFULSWBWCQACUMRSOAVDVLNUNAVMBCVMUQPAURRVQ
+      EUSTTUTVAVB $.
+  $}
+
+  ${
+    $d B z $.  $d A y z $.  $d ph x y z $.  $d ch x y z $.
+    rexlimdvv3d.1 $e |- ( ph ->
+      ( ( x e. A /\ ( y e. B /\ z e. C ) ) -> ( ps -> ch ) ) ) $.
+    $( An extended version of ~ rexlimdvv to include three set variables.
+       (Contributed by Igor Ieskov, 21-Jan-2024.) $)
+    rexlimdvv3d $p |- ( ph -> ( E. x e. A E. y e. B E. z e. C ps -> ch ) ) $=
+      ( wrex cv wcel wa wi expdimp rexlimdvv rexlimdva ) ABFIKEHKCDGADLGMZNBCEF
+      HIASELHMFLIMNBCOJPQR $.
```

Comment:
The `v` was doubled in `rexlimdvv` because there are two set variables.
Here you don't need to both add the `3` and keep the doubled `v`, you can either use a triple `v` or keep the `3` with a single `v`. I would prefer the latter.

Reply:
I usually prefer grouping left first, as in `( ( x e. A /\ y e. B ) /\ z e. C )`, but that's a matter of personal preference.
You could also have simply used the 3-conjunct `( x e. A /\ y e. B /\ z e. C )`.

Reply:
Both items have been fixed. The theorem was renamed to `rexlimdv3d` and now it uses `( x e. A /\ y e. B /\ z e. C )`.